Sources/PostgresNIO/New/PSQLConnection.swift

```diff
@@ -203,7 +200,7 @@ final class PSQLConnection {
         logger[postgresMetadataKey: .connectionID] = "\(connectionID)"
         
         return eventLoop.flatSubmit {
```

Comment:
Can you add a comment here explaining why there's a wrapping `.flatSubmit()` so others don't get confused as I did?